### PR TITLE
report proto change for channel config

### PR DESCRIPTION
**Pull request by khushbu-maurya**
report proto change for channel config

### CODE REVIEW

File: app/grpc/Report.proto [hunk 1]
```diff
@@ -71,22 +71,18 @@ message ReportLog {
...
-message ChannelProperties {
-    string id = 1;
-    repeated ChannelParameters channel_parameters = 2;
-}
-
-message ChannelParameters {
-    repeated ConnectionProperties connectionProperties = 1;
+message Channelparameters {
+  string id = 1;
+  repeated ConnectionProperties channelParameters = 2;
```
Comment (johibkhan2) [inline app/grpc/Report.proto]: pls rename object name from channelParameters to connectionProperties

File: app/grpc/Report.proto [hunk 2]
```diff
@@ -71,22 +71,18 @@ message ReportLog {
 }
 
 message ChannelConnection {
-    repeated ChannelProperties channelProperties = 1;
+    repeated Channelparameters channelProperties = 1;
```
Comment (johibkhan2) [inline app/grpc/Report.proto]: pls rename object name from connectionProperties to channelParameters

File: app/jobs/channelJobs.js [hunk 3]
```diff
@@ -8,12 +8,20 @@ var job = {
...
+                for (let index = 0; index < channel.length; index++) {
+                    var channelObject = {};
+                    channelObject.id = channel[index].id
+                    channelObject.channelParameters = channel[index].channel_parameters.connectionProperties
+                    channelList.push(channelObject);
+                }
+                
             }
             else {
                 return { success: 0, message: "channel not found" };
```
Comment (johibkhan2) [inline app/jobs/channelJobs.js]: remove success: 0 from the code..message is enough

**Comment by johibkhan2** [comment]
please resolve the conflicts

File: app/app.js [hunk 4]
```diff
@@ -96,9 +96,19 @@ app.get('/api/jobLogs/', (req, res) => {
 });
 
 //Channel testing 
-app.post('/api/addChannelConfigs/', function (req, res) {
+app.post('/api/addEmailConfigs/', function (req, res) {
```
Comment (johibkhan2) [inline app/app.js]: we do not need rest apis....please replace with grpc service

File: app/app.js [hunk 5]
```diff
@@ -96,9 +96,19 @@ app.get('/api/jobLogs/', (req, res) => {
...
     if (req.body) {
-        channelJobs.addChannelConfigs(req.body).then(function (result) {
+        channelJobs.addEmailConfigs(req.body.emailConfig).then(function (result) {
+            res.status(result.success === 1 ? 201 : 302).json({ message: result.message })
+        }, function (err) {
+            res.send(err);
```
Comment (johibkhan2) [inline app/app.js]: when error occured please use this format ..{ message: result.message }..aslo inclue response code

File: app/app.js [hunk 6]
```diff
@@ -96,9 +96,19 @@ app.get('/api/jobLogs/', (req, res) => {
...
+        })
+    }
+});
+
+app.post('/api/addTeamConfigs/', function (req, res) {
```
Comment (johibkhan2) [inline app/app.js]: replace reset api with grpc service

File: app/app.js [hunk 7]
```diff
@@ -96,9 +96,19 @@ app.get('/api/jobLogs/', (req, res) => {
...
+    if (req.body) {
+        channelJobs.addTeamConfigs(req.body.teamConfig).then(function (result) {
             res.status(result.success === 1 ? 201 : 302).json({ message: result.message })
         }, function (err) {
             res.send(err);
```
Comment (johibkhan2) [inline app/app.js]: { message: result.message } when exception occured..also inlcude response code

File: app/app.js (hunk 4 above)
Comment (khushbu-maurya) [reply app/app.js]: rest API is needed for unit test case

File: app/app.js (hunk 5 above)
Comment (khushbu-maurya) [reply app/app.js]: okay

File: app/app.js (hunk 7 above)
Comment (khushbu-maurya) [reply app/app.js]: rest API is needed for unit test case

**Pull request merged**